fbgemm_gpu/src/split_embeddings_cache_cuda.cu

```diff
@@ -438,8 +438,7 @@ __global__ __launch_bounds__(kMaxThreads) void lru_cache_find_uncached_kernel(
   // FIXME: __any_sync with mask isn't supported by HIP yet.
   // See https://fburl.com/fvy7j0lq for the similar context.
   // assert false here with https://fburl.com/pfm7enw2
```

Comment:
It looks that `!__any_sync(0xFFFFFFFFFFFFFFFF, found)` solved the issue, but do we need to keep this `FIXME`?

Reply:
The comments are removed.